[chore][receiver/sqlserver] provide one generic configuration in readme (#40851)
open-telemetry/opentelemetry-collector-contrib: receiver/sqlserverreceiver/README.md

## Configuration

> Following is one generic configuration, with that, you can basically have all the logs / metrics exported

Reviewer: ```suggestion
The following is a generic configuration that can be used for the default logs and metrics scraped
```

> by current sqlserver receiver. We also provided some basic explanation on some of the fields. For the detail 

Reviewer: ```suggestion
by the SQL Server receiver. A basic explanation on some of the fields has also been provided. For more
```

> documents, please refer the following section.

Reviewer: ```suggestion
information, please reference the following section.
```

```yaml
sqlserver:
  collection_interval: 10s                     # interval for overall collection
  computer_name: CustomServer
  instance_name: CustomInstance
  username: myusername
  password: mypassword
  server: sqlserver.address
  port: 1433
  top_query_collection:                        # this collection exports the most expensive queries as logs
    enabled: true

>     lookback_time: 60                          # which time window should we look for the top queries

Author: i have check this field is integer instead of duration. maybe we can update this in a separate pr

## Configuration

The following is a generic configuration that can be used for the default logs and metrics scraped
by the SQL Server receiver. A basic explanation on some of the fields has also been provided. For more
information, please reference the following section.

```yaml
sqlserver:
  collection_interval: 10s                     # interval for overall collection
  instance_name: CustomInstance
  username: myusername
  password: mypassword
  server: sqlserver.address
  port: 1433
  events:

>     db.server.query_sample:

Author: Currently, to enable or disable a collection, we have to configure it under `events` rather than within the `collection`-related section. Personally, I find this a bit unintuitive.

Do you think it would be feasible to modify this on mdatagen side, so that enable/disable configuration could live in the same section as the rest of the collection settings?
@sincejune

Reviewer 2: The solution I came up with is to add a parameters section to the metadata file, structured as follows:

```
parameters:
  <param>:
    type: int
    default: <default_value>

events:
  default_event:
    attributes: [...]
    parameters:
      <param>: <value>
```

This approach works well for parameters that will be used within a single collection. However, it may not be ideal for parameters that need to be shared across multiple collections (e.g., metrics or events).

@dmitryax what do you think?

Author: since this is out of the scope, i will leave you guys to resolve the issue and create follow up issues.
